### PR TITLE
ZCS-11605 delegated admin changes

### PR DESCRIPTION
**Pull request by subrahmanyaShetty090**
Added 'certificatesView' to Ui comoponent which will be part of 'Domain Admin' right checkbox

### CODE REVIEW

**Review by k-kato — COMMENTED** [review]
Please revert the changes on ZaAccount.js so that it does not appear in "File changed" page.

File: WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js [hunk 1]
```diff
@@ -1721,7 +1721,7 @@ function(by, val) {
     } else {
         this[ZaAccount.A2_autodisplayname] = "FALSE";
     }
-
+
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js:1724]: revert

Comment (subrahmanyaShetty090) [reply WebRoot/js/zimbraAdmin/accounts/model/ZaAccount.js]: reverted.

File: WebRoot/js/zimbraAdmin/config/settings/ZaSettings.js [hunk 2]
```diff
@@ -442,6 +442,8 @@ ZaSettings.HELP_CENTER_VIEW = "helpCenterView";
 ZaSettings.ALL_UI_COMPONENTS.push({ value: ZaSettings.HELP_CENTER_VIEW, label: ZaMsg.UI_Comp_HelpCenterView });
 ZaSettings.STATUS_PANE = "statusPane";
 ZaSettings.ALL_UI_COMPONENTS.push({ value: ZaSettings.STATUS_PANE, label: ZaMsg.UI_Comp_StatusPane });
+ZaSettings.CERTS_VIEW = "certificatesView";
+ZaSettings.ALL_UI_COMPONENTS.push({ value: ZaSettings.CERTS_VIEW, label: "certificatesView" });
```
Comment (silentsakky) [inline WebRoot/js/zimbraAdmin/config/settings/ZaSettings.js:446]: should label be translated?

**Review by silentsakky — APPROVED** [review]

File: WebRoot/js/zimbraAdmin/config/settings/ZaSettings.js (hunk 2 above)
Comment (subrahmanyaShetty090) [reply WebRoot/js/zimbraAdmin/config/settings/ZaSettings.js]: added translation strings for label.

**Review by sakshirawat0247 — APPROVED** [review]

**Review by k-kato — APPROVED** [review]

**Review by miteshsavani810 — APPROVED** [review]

**Pull request merged**